.changesets/fix_router-external-extensibility-deprecation.md

```diff
@@ -0,0 +1,4 @@
+### Brief but complete sentence that stands on its own ([Issue #ISSUE_NUMBER](https://github.com/apollographql/router/issues/ISSUE_NUMBER))
```

Comment:
Please update the header `###` text :D
We can skip the Issue number if it's not associated with a public issue.